src/components/backend-ai-console.ts

```diff
@@ -995,7 +995,7 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
                 ·
                 <a style="color:forestgreen;" @click="${() => this.showPPAgreement()}">${_t("console.menu.PrivacyPolicy")}</a>
                 ·
-                <a @click="${() => this.splash.show()}">${_t("console.menu.About")}</a>
+                <a @click="${() => this.splash.show()}">${_t("console.menu.AboutBackend.AI")}</a>
```

Comment:
In translation part, dot (.) is used to divide namespaces. How about removing dot from AboutBackend.AI? (to AboutBackendAI)